packages/front-end/services/experiments.ts

```diff
@@ -123,29 +124,37 @@ export function getRisk(
   return { risk, relativeRisk, showRisk };
 }
 
+type Risk = { risk: number } & (
+  | {
+      show: true;
+      relativeRisk: number;
+    }
+  | {
+      show: false;
+    }
+);
```

Comment:
The code logic is to always have `relativeRisk` set when `showRisk` is `true` so I changed the payload to reflect that.